### DISCUSSION TITLE
Sunshine with steamos distros, bazzite in particular

### DISCUSSION THREAD

**Discussion by herbalizer404** [LizardByte/community#449] [Sunshine]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

I'm trying to configure shortcuts in sunshine to switch automatically between gamemode session and desktop mode session with bazzite os, which is booting by default in gamemode while it's used as a htpc in living room. 

### Expected Behavior

Shortcut to switch directly to desktop mode 
and 
Shortcut to switch directly to gamemode

### Host Operating System

Linux

### Operating System Version

bazzite os

### Architecture

64 bit

### Sunshine commit or version

0.23.1

### Package

Linux - rpm

### GPU Type

AMD

### GPU Model

7900 GRE

### GPU Driver/Mesa Version

24.0.5

### Config

```Shell
empty file
```

### Apps

```json
{
    "env": {
        "PATH": "$(PATH):$(HOME)\/.local\/bin"
    },
    "apps": [
        {
            "name": "GAME MODE",
            "image-path": "steam.png",
            "exclude-global-prep-cmd": "",
            "auto-detach": "true",
            "wait-all": "true",
            "exit-timeout": "5",
            "cmd": ""
        },
        {
            "name": "DESKTOP",
            "image-path": "desktop.png",
            "exclude-global-prep-cmd": "",
            "auto-detach": "true",
            "wait-all": "true",
            "exit-timeout": "5",
            "cmd": ""
        }
    ]
}
```

### Relevant log output

```shell
no log
```

**Comment by austonpramodh** [comment]
Did you make any progress on this?

i tried the Desktop Mode, Added "/usr/bin/steamos-session-select plasma" but when its added on prep command, it keeps looping itself. When I add it on detached command, Seems like Sunshine restarts and moonlight looses connection to sunshine.

[a comment by an automated account is not shown]
